cmd/juju/application/consume.go

```diff
@@ -7,46 +7,45 @@ import (
 	"github.com/juju/cmd"
 	"github.com/juju/errors"
 	"gopkg.in/juju/names.v2"
+	"gopkg.in/macaroon.v1"
 
 	"github.com/juju/juju/api/application"
+	"github.com/juju/juju/api/applicationoffers"
 	"github.com/juju/juju/apiserver/params"
 	"github.com/juju/juju/cmd/modelcmd"
 	"github.com/juju/juju/core/crossmodel"
 )
 
 var usageConsumeSummary = `
-Add a remote application to the model.`[1:]
+Add a remote offer to the model.`[1:]
 
 var usageConsumeDetails = `
-Adds a remote application to the model. Relations can be created later using "juju relate".
+Adds a remote offer to the model. Relations can be created later using "juju relate".
```

Comment:
Still worth mentioning that it creates a "remote" application in the model representing the offer? Or does it work ok to just think of it as an offer and create relations to that? (I might be too familiar with the previous terminology to judge that.)